app/controllers/users_controller.rb

```diff
@@ -1,3 +1,28 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash.now[:success] = t ".new.sucess"
```

Comment:
flash và flash.now khác gì nhau em

Reply:
Flash.now nó chỉ hiển thị rồi mất, Flash nó hiển thị mãi  ạ

Reply:
sai rồi em

Reply:
Flash - các giá trị được lưu trữ ở đó sẽ chỉ khả dụng trong yêu cầu tiếp theo, rất hữu ích cho việc truyền thông báo lỗi, hoàn toàn tùy thuộc vào hành động tiếp theo để quyết định.
Flash.now thêm các giá trị vào flash sẽ giúp chúng có sẵn cho yêu cầu tiếp theo, nhưng đôi khi bạn có thể muốn truy cập các giá trị đó trong cùng một yêu cầu.